documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -52,43 +52,43 @@ spec:
         - name: strimzi-topic-operator
           # ...
           env:
-            - name: STRIMZI_NAMESPACE <1>
+            - name: STRIMZI_NAMESPACE # <1>
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-            - name: STRIMZI_KAFKA_BOOTSTRAP_SERVERS <2>
+            - name: STRIMZI_KAFKA_BOOTSTRAP_SERVERS # <2>
               value: my-kafka-bootstrap-address:9092
-            - name: STRIMZI_RESOURCE_LABELS <3>
+            - name: STRIMZI_RESOURCE_LABELS # <3>
               value: "strimzi.io/cluster=my-cluster"
-            - name: STRIMZI_ZOOKEEPER_CONNECT <4>
+            - name: STRIMZI_ZOOKEEPER_CONNECT # <4>
               value: my-cluster-zookeeper-client:2181
-            - name: STRIMZI_ZOOKEEPER_SESSION_TIMEOUT_MS <5>
+            - name: STRIMZI_ZOOKEEPER_SESSION_TIMEOUT_MS # <5>
               value: "18000"
-            - name: STRIMZI_FULL_RECONCILIATION_INTERVAL_MS <6>
+            - name: STRIMZI_FULL_RECONCILIATION_INTERVAL_MS # <6>
               value: "120000"
-            - name: STRIMZI_TOPIC_METADATA_MAX_ATTEMPTS <7>
+            - name: STRIMZI_TOPIC_METADATA_MAX_ATTEMPTS # <7>
               value: "6"
-            - name: STRIMZI_LOG_LEVEL <8>
+            - name: STRIMZI_LOG_LEVEL # <8>
               value: INFO
-            - name: STRIMZI_TLS_ENABLED <9>
+            - name: STRIMZI_TLS_ENABLED # <9>
               value: "false"
-            - name: STRIMZI_JAVA_OPTS <10>
+            - name: STRIMZI_JAVA_OPTS # <10>
               value: "-Xmx=512M -Xms=256M"
-            - name: STRIMZI_JAVA_SYSTEM_PROPERTIES <11>
+            - name: STRIMZI_JAVA_SYSTEM_PROPERTIES # <11>
               value: "-Djavax.net.debug=verbose -DpropertyName=value"
-            - name: STRIMZI_PUBLIC_CA <12>
+            - name: STRIMZI_PUBLIC_CA # <12>
               value: "false"
-            - name: STRIMZI_TLS_AUTH_ENABLED <13>
+            - name: STRIMZI_TLS_ENABLED # <13>
```

Comment:
Is this a backwards incompatible change in the TO? Or was it typo in the docs?

Reply:
My mistake. I confused it with the other variable. This went in as part of this change: https://github.com/strimzi/strimzi-kafka-operator/pull/5201
Reverted.